apps/okreads-e2e/src/specs/search-books.spec.ts

```diff
@@ -16,12 +16,27 @@ describe('When: Use the search feature', () => {
     expect(items.length).toBeGreaterThan(1);
   });
 
-  xit('Then: I should see search results as I am typing', async () => {
+  it('Then: I should see search results as I am typing', async () => {
     await browser.get('/');
     await browser.wait(
       ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okreads')
     );
 
-    // TODO: Implement this test!
+    const input = await $('input[type="search"]');
+
+    await input.sendKeys('ang');
+    const angBookItems = await $$('[data-testing="book-item"]');
+    const angFirstResult = angBookItems[0]?.getWebElement();
+    const angFirstResultTitle = await angFirstResult?.findElement(By.className('book--title')).getText();
+    
+    await input.sendKeys('ular');
+    const angularBookItems = await $$('[data-testing="book-item"]');
+    const angularFirstResult = angularBookItems[0]?.getWebElement();
+    const angularFirstResultTitle = await angularFirstResult?.findElement(By.className('book--title')).getText();
```

Comment:
You don't need to do this. It would be enough to sendKeys and then await for the `book-item` components to appear. Checking the title text isn't necessary either, that would ideally be a test for the component which displays each book (if that were broken out into a separate component...)

Reply:
Thanks for the clarification. I made the relevant updates.